### PR TITLE
IH Drip Update

### PR DESCRIPTION
**Pull request by thevandie**
## About The Pull Request

Adds a few new fashion items for IH, as well as a minor resprite. the IHC also gets a DRIPPY new lighter, pretty much specifically for him.

## Why It's Good For The Game

More player choice and drip is always cool, and the full ballistic helmet looked... bad... I may be in the minority here, but I wholeheartedly believe that this tiny change makes the helmet 30x better

## Testing

I launched the game and everything was as it should be. Zero bugs.

## Changelog
:cl: Vandu-Jr, Watler'Je
add: Added a new Warden Jacket for the Gunny (Partially Watler)
add: Added a new peaked cap for the IHC (Watler)
add: Added a new syndicate lighter for the IHC, based on the oddity sprite but functional.
tweak: tiny sprite change to the full ballistic helmet. Instead of having separate lenses, it's now conjoined.
/:cl:

image
image
image
image

### CODE REVIEW

**Comment by thevandie** [comment]
the fuck

**Comment by thevandie** [comment]
ok it worked this time

File: code/game/objects/items/weapons/cigs_lighters.dm [hunk 1]
```diff
@@ -489,6 +489,13 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	item_state = "zippo"
 	price_tag = 50
 
+/obj/item/flame/lighter/zippo/syndicate
+	name = "\improper Syndicate lighter"
```
Comment (Firefox13) [inline code/game/objects/items/weapons/cigs_lighters.dm:493]:
```suggestion
	name = "\improper syndicate lighter"
```

File: code/modules/clothing/suits/armor.dm [hunk 2]
```diff
@@ -76,6 +76,11 @@
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS|LEGS
 	price_tag = 350
 
+/obj/item/clothing/suit/armor/vest/warden/ironhammer
+	name = "Warden's overcoat"
```
Comment (Firefox13) [inline code/modules/clothing/suits/armor.dm:80]: Half of items having jobs designations are capitalized, the other half is not.
Let's just follow the parent item (capitalized) for this item

**Review by Firefox13 — COMMENTED** [review]
Did you check the new warden's overcoat on a female mob? Does it appear at all, and if so, does it fit properly (no clipping pixels)?

**Comment by Firefox13** [comment]
Also it's Watler, not Walter. I'll fix that for you.

File: code/game/objects/items/weapons/cigs_lighters.dm (hunk 1 above)
Comment (Firefox13) [reply code/game/objects/items/weapons/cigs_lighters.dm:493]: nvm let's leave this capitalized too, scratch this

**Comment by thevandie** [comment]
> Did you check the new warden's overcoat on a female mob? Does it appear at all, and if so, does it fit properly (no clipping pixels)?

Forgor women exist

**Comment by thevandie** [comment]
it works, and yeah no problems

**Comment by ValoTheValo** [comment]
Honestly if you could just make the lighter oddity be a functional lighter that would be better

**Comment by thevandie** [comment]
I am also working on that, but that'll be part of a separate PR. And I prefer having the IHC's version be clean and not rusty.

**Review by Firefox13 — APPROVED** [review]
Code is good, sprites look fine too. Finally bulletproof helmet iteration that looks good.

**Comment by thevandie** [comment]
> Code is good, sprites look fine too. Finally a bulletproof helmet iteration that looks good.

literally 3 pixels were changed collectively
if only we knew it was this easy

**Pull request merged**